reana_cluster/cli/__init__.py

```diff
@@ -0,0 +1,93 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of REANA.
+# Copyright (C) 2017 CERN.
+#
+# REANA is free software; you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation; either version 2 of the License, or (at your option) any later
+# version.
+#
+# REANA is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR
+# A PARTICULAR PURPOSE. See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# REANA; if not, write to the Free Software Foundation, Inc., 59 Temple Place,
+# Suite 330, Boston, MA 02111-1307, USA.
+#
+# In applying this license, CERN does not waive the privileges and immunities
+# granted to it by virtue of its status as an Intergovernmental Organization or
+# submit itself to any jurisdiction.
+"""REANA cluster command line interface."""
+
+import logging
+import os
+import sys
+
+import click
+
+from . import cluster
+
+from ..config import cluster_spec_default_file_path
+
+DEBUG_LOG_FORMAT = '[%(asctime)s] p%(process)s ' \
+                   '{%(pathname)s:%(lineno)d} ' \
+                   '%(levelname)s - %(message)s'
+
+LOG_FORMAT = '[%(levelname)s] %(message)s'
+
+
+class Config(object):
+    """Configuration object to share across commands."""
+
+    def __init__(self):
+        """Initialize config variables."""
+        cluster_url = os.environ.get('REANA_CLUSTER_URL',
+                                     'http://admin.reana.cern.ch')
+
+        logging.info('REANA Cluster URL ($REANA_CLUSTER_URL) is: {}'
+                     .format(cluster_url))
+
+
+@click.group()
+@click.option(
+    '--loglevel',
+    '-l',
+    help='Sets log level',
+    type=click.Choice(['debug', 'info']),
+    default='info')
+@click.option('-f',
+              '--file',
```

Comment:
Should we use the same indentation as in the `--loglevel` option?

Reply:
Maybe. It might be more readable.

I couldn't find where that style is used. In all the click examples I checked the latter style is used.

I will change the syntax in all the files and see how things look like then.